### PR TITLE
added `put` statements to log auto generated pages (original source list)

### PR DESCRIPTION
**Pull request by jhotta**

### CODE REVIEW

File: lib/generate_ja_pages.rb [hunk 1]
```diff
@@ -7,7 +7,9 @@ def generate_ja_pages
 
   missingpages = english_pages-japanese_pages
 
-
+  File.open("autogen_files.log", "w") do |f|
+    f.puts(missingpages)
```
Comment (technovangelist) [inline lib/generate_ja_pages.rb]: can you update this so that it only does it for us and not when jenkins runs?

**Comment by technovangelist** [comment]
is this pr still necessary with pr #1115?

**Comment by irabinovitch** [comment]
@jhotta Did you see Matt's question?

File: lib/generate_ja_pages.rb (hunk 1 above)
Comment (jhotta) [inline lib/generate_ja_pages.rb]: Apology!
I will work on it.

**Pull request merged**